src/geo/geohash.clj

```diff
@@ -194,22 +194,48 @@
 
 (defn- queue [] clojure.lang.PersistentQueue/EMPTY)
 
+(defn- geohashes-intersecting-matches!
+  "Given geohash and shape-relation, modify the matches list as necessary
+   :disjoint    no matches
+   :intersects  current geohash matches, but can't extend assertion to subdivisions
+   :within      same as within
+   :contains    all subdivisions at all valid levels match"
+  [min-level max-level gh shape-relation matches]
+  (case shape-relation
+   ;; gh fully in shape. Add subdivisions of all valid
+   ;; levels into matches
+    :contains
+   (let [start-level (max (significant-bits gh) min-level)
```

Comment:
Could this be pulled out into a function...maybe something like `(defn children-at-levels [gh min-level max-level] ...)`?